packages/wireshark.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'wireshark'
+$category = 'Networking Tools'
+
+$shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
+$shortcut = Join-Path $shortcutDir "Wireshark.lnk"
+$executablePath = Join-Path ${Env:ProgramFiles} "Wireshark\Wireshark.exe" -Resolve
+Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executablePath -RunAsAdmin
```

Comment:
Can we wrap all this in a try/catch in case the `$executablePath` doesn't resolve we'll log it? Also, can you assert that the shortcut was actually created?